src/test-run/index.js

```diff
@@ -178,6 +179,9 @@ export default class TestRun extends Session {
         if (this.pendingRequest)
             this._resolvePendingRequest(command);
 
+        if (command.type === COMMAND_TYPE.wait)
```

Comment:
this should be above. We should not resolve pending request with this command. It should be before 
```js
this.currentCommandCallsite = callsite;
```
or before 
```js
if (this.pendingRequest)
    this._resolvePendingRequest(command);
```

@inikulin Should we reject the `wait` command if there is a `pendingJsError`?